docs/source/topics/plugins.rst

```diff
@@ -34,6 +34,30 @@ If you find yourself in a situation where you feel like you need to do any of th
 .. _core: https://github.com/aiidateam/aiida-core
 .. _registry: https://github.com/aiidateam/aiida-registry
 
+.. _topics:plugins:guidelines:
+
+Guidelines for plugin design
```

Comment:
Thanks, I see your point - I checked the place you suggested, and I see the relation, but I feel it would be a bit lost in this very detailed explanation of the API. It would be good to have a place with simple considerations for plugin design - not just parsers and calcjobs, but eventually more.

I suggest to keep it in the plugins topic for the moment, but with this hierarchy

```
Guidelines for plugin design
============================

CalcJob & Parser plugins
--------------------------
```
Sounds good?